tests/testthat/test-contribution_matrix.R

```diff
@@ -641,7 +641,10 @@ test_that("CalculateContributions() gathers indirect covariate effect contributi
 
 test_that("CreateContributionMatrix() produces correct output for the Donegan example", {
   #Load data
+  donegan_original <- read.csv("Donegan_contribution_matrix_data.csv")
   donegan <- read.csv("Donegan_for_MetaInsight.csv")
+  donegan_ids <- data.frame(Number = 1:3, Label = as.character(1:3))
+  donegan_studies <- unique(donegan$Study)
```

Comment:
These additional lines don't seem to be used. What purpose do they serve?

Reply:
If I comment out either line the test fails. `donegan_ids` is a parameter in `CreateContributionMatrix( )` and `donegan_studies` is used in overwriting `CreateVMatrix( )`.
It's strange because the test passed in the original task. I can only think it's because these were both in my environment.